src/core/lib/experiments/experiments.yaml

```diff
@@ -15,14 +15,6 @@
 # Format of each entry:
```

Comment:
Probably worth a few lines explaining this defines the experiments, and to refer to rollouts.yaml for the rollout state?

Reply:
Fixed.